swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/Store.swift

```diff
@@ -103,6 +103,12 @@ public final class Store: ObservableObject {
     }
   }
 
+  func installSystemExtension() {
```

Comment:
Offers the user another chance to activate the system extension from the Menubar.

Reply:
Why the indirection through the `Store`? Seems unnecessary. The `MenuBar` is already `os(macOS)` so you can easily call it from there directly.

Reply:
Good call, I was just following convention, but that's cleaner.